lib/view_component/codemods/v3_slot_setters.rb

```diff
@@ -0,0 +1,183 @@
+# Usage (in rails console):
+#
+# Make sure all your ViewComponents are loaded before running the codemod.
+#
+#     Zeitwerk::Loader.eager_load_all
+#
+# Run the codemod:
+#
+#     ViewComponent::Codemods::V3SlotSetters.new.call
+#
+# If your app uses custom paths for views, you can pass them in:
```

Comment:
Could we load these from the rails config instead of requiring them to be passed in?